examples/reminderbot/credentials.yml

```diff
@@ -0,0 +1,35 @@
+# This file contains the credentials for the voice & chat platforms
+# which your bot is using.
+# https://rasa.com/docs/rasa/user-guide/messaging-and-voice-channels/
+
+rest:
+#  # you don't need to provide anything here - this channel doesn't
+#  # require any credentials
+
+callback:
+  # URL to which Core will send the bot responses
+  url: "http://localhost:5034/bot"
```

Comment:
which port is `5034`?

Reply:
I just copy-pasted from the docs: https://rasa.com/docs/rasa/user-guide/connectors/your-own-website/#callbackinput

Reply:
Mhm, maybe let's also link the docs in the comment ?

Reply:
@wochinge Is that ok? Then I'd click "Squash and Merge"

Reply:
Oh. Your message didn't appear until now. Sorry. Will do that.